src/openrct2/drawing/Image.cpp

```diff
@@ -164,8 +164,10 @@ static void FreeImageList(uint32_t baseImageId, uint32_t count)
     Guard::Assert(baseImageId >= BASE_IMAGE_ID, GUARD_LINE);
 
 #ifdef DEBUG
```

Comment:
That's also silly to do, as it is often defined - check for the value instead

Reply:
`DEBUG` and `NDEBUG` are often defined with no value, using `defined(...)` or `ifdef` is fine. The values are never checked anywhere else in our codebase.